mkosi/__init__.py

```diff
@@ -3122,7 +3122,11 @@ def install_gentoo(
         # Please don't move, needs to be called before installing dracut
         # dracut is part of gentoo_pkgs_boot
         configure_dracut(args, packages={"dracut"}, root=root)
-        gentoo.invoke_emerge(args, root, pkgs=gentoo.pkgs_boot)
+        # Installing sys-kernel/installkernel-systemd-boot fails spectacularly
```

Comment:
"spectacularly" is a bit harsh, don't you think? ;)

Reply:
Eeh... I may have been a little annoyed because building a gentoo osi takes some time and it always barfed on me in the last steps 😅